packages/testlab/src/client.ts

```diff
@@ -26,22 +26,29 @@ export function createClientForHandler(
   return supertest(server);
 }
 
+export interface RestServer {
+  options?: {
+    http?: {
+      port?: number;
+    },
+  };
+  start() : Promise<void>;
```

Comment:
Isn't `start()` common to all servers? I think it should go to `interface Server`.